oak_functions/proto/server.proto

```diff
@@ -16,17 +16,25 @@
 
 syntax = "proto3";
 
-package oak.remote_attestation;
+package oak.functions.server;
 
 option java_multiple_files = true;
-option java_package = "oak.remote_attestation";
+option java_package = "oak.functions.server";
+
+// Wrapper around the encrypted payload.
+// TODO(#2144): Extend this with additional potentially public fields, for instance for specifying
+// policies.
+message SecureRequest {
```

Comment:
I don't really like the "secure" name in this context, feel free to submit but let's keep thinking about this.

cc @project-oak/core in case anyone has any suggestion

Reply:
`RequestWithEncryptedPayload` is more accurate, but that is very long name.

Reply:
I think we should try to rename this to just `Request` eventually, and find some other name for `AttestedInvokeRequest`, e.g. something with `wrapper` or `frame` or `handshake` or something else.

Reply:
Done.